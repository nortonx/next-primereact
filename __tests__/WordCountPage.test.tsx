import { render, screen } from  "@testing-library/react"
import WordCountPage from "@/app/word-count/page"

describe("Wound Count Page", () => {
  it("should render page and match snapshot", async () => {
    const { container } = render(<WordCountPage/>)
    const page = await screen.findByTestId("word-count-page");
    expect(page).toBeInTheDocument()
    expect(container).toMatchSnapshot()
  })
})