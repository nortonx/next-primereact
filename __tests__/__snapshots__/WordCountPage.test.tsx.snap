// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wound Count Page should render page and match snapshot 1`] = `
<div>
  <section
    class="p-3 surface-section text-center"
    data-testid="word-count-page"
  >
    <h1
      class="text-900 font-bold text-6xl"
    >
      Word Counter
    </h1>
    <div
      class="input-text flex justify-content-center"
    >
      <textarea
        class="w-6 p-inputtextarea p-inputtext p-component p-inputtextarea-resizable w-6"
        data-pc-name="inputtextarea"
        data-pc-section="root"
        data-testid="word-count-textarea"
        rows="10"
        style="overflow: hidden; height: 0px;"
      />
    </div>
    <div
      class="counters flex gap-2 justify-content-around mt-3"
    >
      <div
        class="w-3 p-card p-component"
        data-pc-name="card"
        data-pc-section="root"
        data-testid="words"
      >
        <div
          class="p-card-body"
          data-pc-section="body"
        >
          <div
            class="p-card-title"
            data-pc-section="title"
          >
            Words
          </div>
          <div
            class="p-card-content"
            data-pc-section="content"
          >
            <span
              class="text-xl font-medium"
              data-testid="words-value"
            >
              0
            </span>
          </div>
        </div>
      </div>
      <div
        class="w-3 p-card p-component"
        data-pc-name="card"
        data-pc-section="root"
        data-testid="characters"
      >
        <div
          class="p-card-body"
          data-pc-section="body"
        >
          <div
            class="p-card-title"
            data-pc-section="title"
          >
            Characters
          </div>
          <div
            class="p-card-content"
            data-pc-section="content"
          >
            <span
              class="text-xl font-medium"
              data-testid="characters-value"
            >
              0
            </span>
          </div>
        </div>
      </div>
      <div
        class="w-3 p-card p-component"
        data-pc-name="card"
        data-pc-section="root"
      >
        <div
          class="p-card-body"
          data-pc-section="body"
        >
          <div
            class="p-card-title"
            data-pc-section="title"
          >
            Sentences
          </div>
          <div
            class="p-card-content"
            data-pc-section="content"
          >
            <span
              class="text-xl font-medium"
            >
              0
            </span>
          </div>
        </div>
      </div>
      <div
        class="w-3 p-card p-component"
        data-pc-name="card"
        data-pc-section="root"
      >
        <div
          class="p-card-body"
          data-pc-section="body"
        >
          <div
            class="p-card-title"
            data-pc-section="title"
          >
            Paragraphs
          </div>
          <div
            class="p-card-content"
            data-pc-section="content"
          >
            <span
              class="text-xl font-medium"
            >
              0
            </span>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
